java/arcs/core/data/testdata/BUILD

```diff
@@ -11,11 +13,15 @@ filegroup(
         ["*"],
         exclude = ["BUILD"],
     ),
-    visibility = ["//visibility:public"],
 )
 
 arcs_kt_gen(
     name = "example_gen",
     srcs = ["WriterReaderExample.arcs"],
-    visibility = ["//visibility:public"],
 )
+
+# TODO: It doesn't resovle currently, can we investigate why?
```

Comment:
Let's link this to a bug so we can keep track of the issues (likely, with sub-bugs).

Reply:
Done :)